### PR TITLE
(PC-11884) fix(LogoMinistere): suppression marge à gauche et correction page profil web mobile 

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-11884

Also include : [(PC-11889)](https://passculture.atlassian.net/browse/PC-11884)

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- x ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     | image | image | image |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/profile/pages/Profile.tsx [hunk 1]
```diff
@@ -105,148 +105,150 @@ export const Profile: React.FC = () => {
...
-          {!!isLoggedIn && (
-            <React.Fragment>
-              <Row
-                title={t`Informations personnelles`}
-                type="navigable"
-                onPress={() => navigate('PersonalData')}
-                icon={ProfileIcon}
-                style={styles.row}
-              />
+    <React.Fragment>
```
Comment (antoinewg) [inline src/features/profile/pages/Profile.tsx:108]: le Fragment sert à rien et complexifie la relecture :)

Comment (kopax-polyconseil) [reply src/features/profile/pages/Profile.tsx:108]: pardon c'était un test, merci :)

[the remark this answers is not in the document]
Comment (kopax-polyconseil) [reply src/features/profile/pages/Profile.tsx:108]: pardon c'était un test, merci :)

**Review by antoinewg — APPROVED** [review]

File: src/features/profile/pages/Profile.tsx [hunk 2]
```diff
@@ -271,9 +271,9 @@ const Row = styled(SectionRow).attrs({
   iconSize: SECTION_ROW_ICON_SIZE,
 })``
 
-const BottomSpacing = styled.View({
-  paddingBottom: TAB_BAR_COMP_HEIGHT + getSpacing(2),
-})
+const BottomSpacing = styled.View(({ theme }) => ({
+  height: theme.showTabBar ? theme.tabBarHeight + getSpacing(2) : 0,
+}))
```
Comment (BenStirrup) [inline src/features/profile/pages/Profile.tsx:276]: ca va pas le coup de mettre cette modif dans `useCustomSafeInsets` ou `Spacer.BottomScreen` ?

Comment (kopax-polyconseil) [reply src/features/profile/pages/Profile.tsx]: J'ai maj et j'ai rajouté une props à `TabBarSpacer` pour désactiver la hauteur.

**Review by BenStirrup — APPROVED** [review]

**Pull request merged**